tests/Concerns/WithCustomCsvSettingsTest.php

```diff
@@ -54,6 +54,7 @@ public function getCsvSettings(): array
                     'use_bom'                => true,
                     'include_separator_line' => true,
                     'excel_compatibility'    => false,
+                    'output_encoding'        => 'UTF-8',
```

Comment:
I think we have to default this to an empty string to make it fully backwards compatible. `''` is the default within the Csv writer. If left empty, it won't do any convert encoding when writing.

Reply:
Yes, you are absolutely right.